Formula/p/python-matplotlib.rb

```diff
@@ -0,0 +1,67 @@
+class PythonMatplotlib < Formula
+  desc "Python library for creating static, animated, and interactive visualizations"
+  homepage "https://matplotlib.org/"
+  url "https://files.pythonhosted.org/packages/fb/ab/38a0e94cb01dacb50f06957c2bed1c83b8f9dac6618988a37b2487862944/matplotlib-3.8.2.tar.gz"
+  sha256 "01a978b871b881ee76017152f1f1a0cbf6bd5f7b8ff8c96df0df1bd57d8755a1"
+  license "PSF-2.0"
+
+  depends_on "meson" => :build
+  depends_on "meson-python" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "pybind11" => :build
+  depends_on "python-build" => :build
+  depends_on "python-certifi" => :build
+  depends_on "python-setuptools-scm" => :build
+  depends_on "fonttools"
+  depends_on "freetype"
+  depends_on "numpy"
+  depends_on "pillow"
+  depends_on "python-cycler"
+  depends_on "python-dateutil"
+  depends_on "python-kiwisolver"
+  depends_on "python-packaging"
+  depends_on "python-pyparsing"
+  depends_on "python@3.12"
+  depends_on "qhull"
+  depends_on "six"
+
+  on_linux do
+    depends_on "patchelf" => :build
```

Comment:
do you remember what uses this? I vaguely remember this from another formulae that uses `matplotlib` but it was dynamically added only if some other requirement (maybe `ninja?) wasn't found

Reply:
It's because of `meson-python`. [The changelog for version 0.15.0](https://meson-python.readthedocs.io/en/latest/changelog.html#id1) makes it clear that this is always required for any project that uses `meson-python` on Linux.

Reply:
ah ok thanks, so they just changed it